src/System.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UDTs/Circle/Circle.csproj

```diff
@@ -1,7 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <TargetFramework>$(NetMinimum)</TargetFramework>
```

Comment:
Is this tfm correct?

Reply:
Yes, we only run tests on .NET Core targets where the driver code actually applies. 
They are not run on .NET Framework, since this we type-forward to System.Data.dll for .NET Framework.

Reply:
The tfms for all test projects needed the -unix or -windows suffix in order for TargetPlatformIdentifier to get a proper value in the inner builds.